src/main/java/uk/gov/justice/digital/hmpps/cmd/api/client/CsrClient.kt

```diff
@@ -61,6 +62,8 @@ class CsrClient(val regionData: Regions, @Value("\${jwt.secret}") val secret: St
             log.info("Found 0 shift notifications, PlanUnit $planUnit, Region $region")
             return listOf()
         }
+        notifications.shiftTaskNotifications.forEach { notification -> notification.shiftType = if(notification.shiftType.equals(ShiftNotificationType.SHIFT.value, true)) { ShiftNotificationType.SHIFT_TASK.value } else { ShiftNotificationType.OVERTIME_TASK.value } }
+
         return notifications.shiftTaskNotifications
```

Comment:
The shift task notification stored proc sends SHIFT and OVERTIME, but we can replace them here to get a more specific user message.